data/abilities.js

```diff
@@ -100,8 +100,7 @@ exports.BattleAbilities = {
 		onBasePowerPriority: 8,
 		onBasePower: function (basePower, pokemon) {
 			let boosted = true;
-			let allActives = pokemon.side.active.concat(pokemon.side.foe.active);
-			for (const target of allActives) {
+			for (const target of pokemon.side.active.concat(pokemon.side.foe.active)) {
```

Comment:
I would definitely either do a double `for`-loop, or I would at least define `allActives` on its own line; it's too long for me to like it being inline.